plugins/ci/skills/analyze-disruption/find_disruption_runs.py

```
@@ -527,9 +546,39 @@ def main(argv=None):
+
     since_ms = int((time.time() - args.since_hours * 3600) * 1000)
-    filter_dict = build_sippy_filter(variants, since_ms)
-    rows = fetch_runs(release, filter_dict, args.limit)
+
+    multi_keys = [(k, v.split(",")) for k, v in variants.items() if "," in v]
+    if multi_keys:
+        seen_prow_ids = set()
+        rows = []
+        keys = [k for k, _ in multi_keys]
+        vals = [v for _, v in multi_keys]
+        variant_combos = [dict(zip(keys, combo)) for combo in itertools.product(*vals)]
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)